app/globals.css

```
@@ -8,7 +8,8 @@
   --color-background: var(--background);
   --color-foreground: var(--foreground);
   --font-sans: var(--font-inter);
-  --font-mono: ui-monospace, SFMono-Regular, Menlo, monospace;
+  --font-mono: var(--font-mono), ui-monospace, Menlo, monospace;
```

Stylelint (17.9.0)

[error] Expected "Menlo" to be "menlo" (value-keyword-case)

(value-keyword-case)